### PR TITLE
2996513 scope lock fix

### PR DESCRIPTION
**Pull request by jimwills123**

### CODE REVIEW

File: js_manager.install [hunk 1]
```diff
@@ -85,9 +92,29 @@ function js_manager_schema() {
   return $schema;
 }
 
+function js_manager_schema_alter() {
```
Comment (davidgrayston) [inline js_manager.install]: Is it necessary to alter the schema provided by this module?

File: js_manager.install [hunk 2]
```diff
@@ -85,9 +92,29 @@ function js_manager_schema() {
...
 function js_manager_uninstall() {
   drupal_uninstall_schema('js_manager_add_js');
 }
+
+/**
+ * Adds scope_lock option to JS scripts
+ */
+function js_manager_update_7001() {
+   $schema = drupal_get_schema('js_manager_add_js');
+  db_add_field('js_manager_add_js', 'scope_lock', $schema['fields']['scope_lock']);
```
Comment (davidgrayston) [inline js_manager.install]: Inconsistent indentation

**Review by davidgrayston — CHANGES_REQUESTED** [review]
Few very minor things, but otherwise looks like a good solution to prevent advagg moving head scripts to footer

File: js_manager.install (hunk 1 above)
Comment (jimwills123) [reply js_manager.install]: done

File: js_manager.install (hunk 2 above)
Comment (jimwills123) [reply js_manager.install]: done

**Review by jon-edery — APPROVED** [review]

**Pull request merged**